scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -2576,12 +2675,14 @@ def get_user_agent_fragment(self):
         k8s_cache = self.__get_k8s_cache()
         ver = None
         runtime = None
-        if k8s_cache:
-           ver = k8s_cache.get_api_server_version()
-           runtime = k8s_cache.get_container_runtime()
+        if k8s_cache and not self._global_config.k8s_use_controlled_warmer:
+            # echee TODO: currently, if k8s_use_controlled_warmer is True, version checks are disabled.
```

Comment:
There are some outstanding TODOs that must be resolved before merging back to master. This is one of them.